server/src/resolvers.js

```diff
@@ -4,16 +4,150 @@ const {User} = require('./resolvers/user');
+          title: 'Introduction to Algorithm',
+          description: 'Basic to ICPC World Final... You think you can handle this?',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 2,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'What is meaning of life?',
+          description: 'Sometimes we thinking about huge size of the universe. Compare to what humanity accomplished so far, the universe seems wast of space... isn\' it?',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 3,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'If AI gets self awareness...',
+          description: 'That means bad thing to human? YES. But in a perspective of evolution? NO.',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 4,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'When Space X going to launch a rocket to the Mars?',
+          description: 'Let\'s analyse the probability!',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 5,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'title1',
+          description: 'description1',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 6,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'title2',
+          description: 'description2',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 7,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'title3',
+          description: 'description3',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 8,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'title4',
+          description: 'description4',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        }
+      ];
+
+      let ret = [];
+      if(!pageSize) {
+        ret = events_list;
+      }else{
+        for(let j = 0; j < Math.min(pageSize, 9); j++){
```

Comment:
In general, do not use "j" as an array index unless you're in a nested for loop where the outer loop uses i as the index. This makes it look like there's an outerloop which is confusing.